verticals/vertical-crm/commonModels.ts

```diff
@@ -15,19 +24,89 @@ export const contact = zBaseRecord
 
 export const lead = zBaseRecord
   .extend({
+    id: z.string().nullish(),
     name: z.string().nullish(),
+    firstName: z.string().nullish(),
```

Comment:
Let's make sure to `snake_case` as that is the convention used for API requests generally.